infrastructure/conf/nginx-conf-deploy/default

```diff
@@ -33,6 +33,15 @@ server {
     location = /fr/applicant/dashboard { return 301 /fr/applicant/profile-and-applications$is_args$args; }
     location = /en/privacy-notice { return 301 /en/privacy-policy$is_args$args; }
     location = /fr/privacy-notice { return 301 /fr/privacy-policy$is_args$args; }
+    location = /en/talent/profile { return 301 /en/applicant$is_args$args; }
```

Comment:
These two are currently the fallback landing page for login redirect.  You should probably update that in this PR, too.  It will require a deployment step to update the value in Azure, as well.
https://github.com/GCTC-NTGC/gc-digital-talent/blob/9f1d5c16acb1fc20bdf6c290e97528acfec80308/api/app/Http/Controllers/AuthController.php#L122

Reply:
oh yeah, thanks for nothing that :+1: Updated in [update login redirect path](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10304/commits/937e2f178eb5e755eaceb6e90330f7152aabb3ed)